Audio/BnkCompiler/Compiler.cs

```diff
@@ -16,24 +19,282 @@ public class CompileResult
         public CompilerData Project { get; set; }
         public PackFile OutputBnkFile { get; set; }
         public PackFile OutputDatFile { get; set; }
-        public PackFile NameList { get; set; }
+        public PackFile OutputStatesDatFile{ get; set; }
     }
 
     internal static class CompilerConstants
     {
         public static readonly string Game_Warhammer3 = "Warhammer3";
-        public static readonly string Language_English = "English(uk)";
-        public static readonly string Mixers_Diplomacy = "54848735";
+        public static readonly Dictionary<string, uint> VanillaIds = new Dictionary<string, uint>()
+        {
+            // Ids for Event mixers
+
+            {"advisor", 517250292},
+            {"battle individual vocalisation", 508226369},
+            {"diplomacy line", 54848735},
+            {"ind magic", 140075115},
+            {"movie", 573597124},
+            {"quest battle", 659413513},
+            {"ui", 608071769},
+            {"event narration", 517250292},
+
+            // Ids for Dialogue Event mixers 
+            {"battle_vo_conversational", 600762068},
+            {"battle_vo_orders", 1009314120},
+            {"campaign_vo_conversational", 652491101},
+            {"campaign_vo", 306196174},
+            {"frontend_vo", 745637913},
+
+            // Ids for attenuation objects for Dialogue Events 
+            {"battle_vo_conversational_attenuation", 649943956},
+            {"battle_vo_orders_attenuation", 803409642},
+            {"campaign_vo_conversational_attenuation", 62329658},
+            {"campaign_vo_attenuation", 432982952},
+        };
+
+        public static string MatchDialogueEventToBnk(string dialogueEvent)
```

Comment:
Not sure I understand what this function does?

Reply:
It is a way of getting the parent bnk of a dialogue event which is used here to get the attenuation ID from the dictionary of vanilla IDs:
```
                var dialogueEventBnk = CompilerConstants.MatchDialogueEventToBnk(inputSound.DialogueEvent);
                var attenuationKey = $"{dialogueEventBnk}_attenuation";
```